ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java

```diff
@@ -15,7 +15,7 @@
     /**
      * Check the list of modules to be sure they are all started.
      */
-    @Ignore
+    @Test
     @Category(org.openmrs.reference.groups.BuildTests.class)
```

Comment:
for some reason i have made this change two times and committed but it stays .  i don't know why

Reply:
```
    @Test
    @Category(BuildTests.class) 
    public void checkModules() throws Exception {
```
i did this and pushed but the change doesn't appear

Reply:
@sherrif10  do you have an idea how i can fix this